### ISSUE TITLE
Pre-trained Weight Decay

### ISSUE THREAD

**Issue by Yupei-Du** [asappresearch/revisit-bert-finetuning#4]
Hi, thanks for the great paper and implementation. I have a question regarding pre-trained weight decay. 

Assume I don't want to use layerwise learning rate decay (```args.layerwise_learning_rate_decay == 1.0```), in ```get_optimizer_grouped_parameters``` I will get two parameter groups: decay and no decay. Hence in ```PriorWD``` all initial parameters will be recorded 
```python
self.prior_params = {}
        for i, group in enumerate(self.param_groups):
            for p in group["params"]:
                self.prior_params[id(p)] = p.detach().clone()
```
In this case, during updating, all parameters will have a weight decay towards the initial value. 
However, I suppose that we don't want the classification layer to decay towards the init values, so it seems that we should really create a separate parameter group for the classification layer. 

I was wondering whether my reasoning would make sense to you, or where did I get it wrong?
